test/gui/drone/comment.sh

```diff
@@ -22,4 +22,12 @@ else
         echo "creating comment for server log"
         echo "Server Logs: (${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/serverlog.log)" >> $1/comments.file
     fi
+
+    if ! [[ $(find $1/screenshots -maxdepth 0 -empty) ]]; then
+        echo "creating comment for screenshots"
+        echo "Screenshots:" >> $1/comments.file 
+        for i in $(ls $1/screenshots); do
+            echo "- [$i](${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/guiReportUpload/screenshots/$i)" >> $1/comments.file
```

Comment:
```suggestion
            echo "- [$i](${CACHE_ENDPOINT}/${CACHE_BUCKET}/$2/$3/${GUI_TEST_REPORT_DIR}/screenshots/$i)" >> $1/comments.file
```
also $1 $2 $3 should be given nice names
both could happen in a different PR

Reply:
$GUI_TEST_REPORT_DIR is `/drone/src/test/guiReportUpload`. We either need to create new env variable to store the directory name `guiReportUpload` or we should leave it as it is, AFAIK.

And yes, we can give some nice name to all the variables, that can be done in some other PR.